styles/mixins/_atoms.scss

```diff
@@ -0,0 +1,35 @@
+@use 'sass:color';
+@use 'sass:list';
+
+@use '../variables' as var;
+
+$-color-border: var.$color-border;
+$-color-shadow: var.$color-shadow;
+
+/**
+ * Add border styles for all element sides (default) or a specific single side
+ *
+ * @param {'left'|'right'|'bottom'|'top'} [$side]
+ */
+@mixin border($side: null) {
+  $valid-sides: left, right, bottom, top;
+  $border-property: border;
+  @if list.index($valid-sides, $side) {
+    $border-property: border-#{$side};
+  }
+  #{$border-property}: 1px solid $-color-border;
+}
+
+/**
+ * Style a shadow off of the element. `$active` variant is larger, darker,
+ * intended for hovered/active elements.
+ *
+ * @param {boolean} [$active]
+ */
+@mixin shadow($active: false) {
```

Comment:
Apologies for my ignorance of sass. What happens if one uses like this: `shadow(left)`?

Reply:
Don't apologize! There's no reason you should have the entire SASS API within your brain! I will check this out; it depends on whether SASS interprets the string value of `left` as truthy.

Reply:
Checked: string values are considered truthy.